.gitignore

```diff
@@ -157,6 +159,8 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+# Database
+mercari.sqlite3
 # PyCharm
 #  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
```

Comment:
also adding `.idea` might be a good here. we don't want local changes like IDE configuration to be committed accidentally.

Reply:
so .idea is the configuration file of IDE, right?